spec/generators/clearance/password_reset_migration/password_reset_migration_generator_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+require "generators/clearance/password_reset_migration/password_reset_migration_generator"
```

Comment:
We could either ignore this or rename this generator. I have no particular preference.